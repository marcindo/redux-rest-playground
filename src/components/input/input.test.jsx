import React from 'react';
import renderer from 'react-test-renderer';

import Input from '.';

it('Renders btn', () => {
  ['Lorem', 'Ipsum'].forEach((text) => {
    const component = renderer.create(<Input>{text}</Input>);

    expect(component).toMatchSnapshot();
  });
});
