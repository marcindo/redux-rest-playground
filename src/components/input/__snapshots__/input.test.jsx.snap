// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders btn 1`] = `
<input
  className="sc-bdVaJa crseKW"
  size={8}
  type="text"
>
  Lorem
</input>
`;

exports[`Renders btn 2`] = `
<input
  className="sc-bdVaJa crseKW"
  size={8}
  type="text"
>
  Ipsum
</input>
`;
